src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/ParenPadCheckTest.java

```diff
@@ -154,6 +154,14 @@ public void testTrimOptionProperty() throws Exception {
                expected);
     }
 
+    @Test
+    public void testUpperCaseProperty() throws Exception {
+        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
+        verifyWithInlineConfigParser(
+                getPath("InputParenPadToCheckUpperCaseFunctionInOptionProperty.java"),
+               expected);
```

Comment:
I noticed this in this PR and previous. `expected` is indented 1 less then where it should.

Please ensure they line up. Please fix previous test in this same PR.